src/views/Regions/Regions.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from 'react';
+import { fetchPokedexList, fetchRegionList } from '../../services/pokemon';
+import { Link } from 'react-router-dom';
+import { slicePokedex } from '../../utils/utils';
+
+import styles from './Regions.css';
+
+export default function Regions() {
+  const [pokedexes, setPokedexes] = useState([]);
+  const [regions, setRegions] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  // Get Pokedexes
+  useEffect(() => {
+    const getPokedexes = async () => {
+      const pokedexList = await fetchPokedexList();
+      const regionList = await fetchRegionList();
+      setPokedexes(pokedexList);
+      setRegions(regionList);
+      setLoading(false);
+    };
+    getPokedexes();
+  }, []);
+
+  if (loading) return <p>Loading...</p>;
+
+  const filterRegions = regions.map((region) => {
```

Comment:
I might consider doing this inside of your fetch, it would allow you to only have a single state in this component in addition to loading